sjb/config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install.yml

```diff
@@ -2,10 +2,9 @@
 parent: 'test_cases/test_branch_origin_extended_conformance_install.yml'
 overrides:
   junit_analysis: True
-  sync_repos:
-    - name: "aos-cd-jobs"
-    - name: "openshift-ansible"
-      type: "pull_request"
-    - name: "origin"
-    - name: "image-registry"
-    - name: "kubernetes-metrics-server"
+  sync:
```

Comment:
This seems unrelated to the PR, fine with it if you are, just don't know if it was intentional.

Reply:
These have to change since this PR conflates a) the migration to a parent and b) the migration from `sync_repos` to `sync`